### PR TITLE
Add Popper component

### LINKED ISSUE #71: ✨ Component: Popper
- [ ] Implementation
- [ ] Example
- [ ] Animation
- [ ] Storybook/Histoire
- [ ] Tests
- [ ] Docs

Internal component used in all the overlay components using floating-ui

### PR DESCRIPTION
**Pull request by HalfdanJ**
Fixes #71 

I ended up not using `svelte-floating-ui`, but instead just using `@floating-ui/core`. The svelte library is first of all a very light wrapper, and second a bit too opinionated about how styles are applied.

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,38 +1,51 @@
 <script lang="ts">
 	import { getPropsObj } from './(previews)/helpers';
-	import schemas from './(previews)/schemas';
+	import { internal, components } from './(previews)/schemas';
+
+	const categories = {
```
Comment (TGlide) [inline src/routes/+page.svelte:5]: Nice

**Review by TGlide — APPROVED** [review]
Nice work! Wdyt of the structure I created on the dialog draft PR? In which I have a folder called internal, that contains helpers and components? I also did some changes to ReactiveContext so we aren't obligated to use a setterFn on values.

**Comment by HalfdanJ** [comment]
@TGlide this should be ready now

**Comment by TGlide** [comment]
a

**Pull request merged**